src/Nox.Types/Types/Color/Color.cs

```diff
@@ -0,0 +1,213 @@
+    public static Color From(byte red, byte green, byte blue)
+    {
+        var color = System.Drawing.Color.FromArgb(byte.MaxValue, red, green, blue);
+
+        return CreateColor(color);
+    }
+
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object.
+    /// </summary>
+    /// <param name="alpha">The alpha color.</param>
+    /// <param name="red">The red color.</param>
+    /// <param name="green">The green color.</param>
+    /// <param name="blue">The blue color.</param>
+    /// <exception cref="TypeValidationException"></exception>
+    public static Color FromRgba(byte alpha, byte red, byte green, byte blue)
+    {
+        var color = System.Drawing.Color.FromArgb(alpha, red, green, blue);
+
+        return CreateColor(color);
+    }
+
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object
+    /// </summary>
+    /// <param name="name">The origin value to create the <see cref="Color"/> with</param>
+    /// <returns></returns>
+    /// <exception cref="TypeValidationException"></exception>
+    public static Color FromName(string name)
+    {
+        var color = System.Drawing.Color.FromName(name);
+
+        return CreateColor(color);
+    }
+
+    /// <summary>
+    /// Creates a new instance of <see cref="Color"/> object
+    /// </summary>
+    /// <param name="alphadecimal">The origin value to create the <see cref="Color"/> with</param>
+    /// <returns></returns>
+    /// <exception cref="TypeValidationException"></exception>
+    public static Color FromAlphaColor(string alphadecimal)
+    {
+        return ColorConverter.ConvertFromString(alphadecimal, CultureInfo.InvariantCulture);
+    }
+
+    /// <summary>
+    /// Returns the bytes representation of the color value
+    /// </summary>
+    /// <returns>The byte representation</returns>
+    public byte[] ToBytes() => new byte[] { _alpha, _red, _green, _blue };
+
+    /// <summary>
+    /// Returns the string representation of the color value, formatted in Hexadecimal format string
+    /// </summary>
+    /// <returns>The string representation</returns>
+    public string ToHexa() => $"#{_alpha:X2}{_red:X2}{_green:X2}{_blue:X2}";
+
+    /// <summary>
+    /// Returns the string representation of the color value, formatted in Hex format string
+    /// </summary>
+    /// <returns>The string representation</returns>
+    public string ToHex() => $"#{_red:X2}{_green:X2}{_blue:X2}";
+
+    /// <summary>
+    /// Returns the string representation of the color value, formatted in name format string
+    /// </summary>
+    /// <returns>The string representation</returns>
+    public string ToName()
+    {
+        var color = System.Drawing.Color.FromArgb(_alpha, _red, _green, _blue);
+
+        var knownColor = KnownColor.Colors.FirstOrDefault(kc => kc.Value.ToArgb() == color.ToArgb());
+
+        return knownColor.Key ?? "UnknownColor";
+    }
+
+    /// <inheritdoc />
+    public override string ToString()
```

Comment:
We do not need this overriding of the method as it should already work like that with the Base implementation, please add a test for this.

Reply:
Done